imports/api/methods.js

```diff
@@ -20,7 +20,7 @@ Meteor.methods({
     return true;
   },
   saveJob: function (vacancy) {
-    Jobs.insert(vacancy);
+    return Jobs.insert(vacancy);
```

Comment:
this return statement allows it to return the job ID in case of success, we are using this ID to redirect to the view that shows this job after creating it